.pre-commit-hooks.yaml

```diff
@@ -1,3 +1,11 @@
+- id: pdm-check
+  name: pdm-check
```

Comment:
Can you make the name `pdm-lock-check` to be more specific?

Reply:
https://github.com/pdm-project/pdm/pull/1475/commits/b0e5e824297ea5d1b14760d1d2d00e90a0e50c6d